NewSpring/ApollosProject
# [DEV-3738] Comms Onboarding Updates

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds updates from the Comms team for onboarding. These include:
- Text on every onboarding page.
- Updated images for personalization, location, and notifications

### How do I test this PR?
Open up your sim and go through onboarding. Make sure nothing looks out of place or broken.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
